src/functionalTest/resources/feature-toggle-enabled.json

```diff
@@ -0,0 +1,5 @@
+{
+  "description": "Create feature for smoke test",
```

Comment:
Is this a description of the action or the actual toggle?

Reply:
description of feature toggle